.github/workflows/ci.yaml

```
@@ -39,6 +51,59 @@ jobs:
 
       - run: go build -o temporal-features
 
+      - name: Get the latest release version
+        id: latest_version
+        run: |
+          latest_version=$(./temporal-features latest-sdk-version --lang go)
+          go_latest=$(echo "$latest_version" | sed 's/^v//')
+          if [ "${{ github.event.inputs.go_sdk_version }}" != "" ]; then
+            go_latest="${{ github.event.inputs.go_sdk_version }}"
+            echo "User-provided Go SDK version: $go_latest"
+          else
+            echo "Latest Go SDK release version: $go_latest"
+          fi
+          echo "go_latest=$go_latest" >> $GITHUB_OUTPUT
```

May want to default to input, remove the thing removing the `v` from output (you already do that in the cmd), any may not need to log if it was a workflow parameter, e.g.:

```suggestion
          go_latest="${{ github.event.inputs.go_sdk_version }}"
          if [ -z "$value" ]; then
            go_latest=$(./temporal-features latest-sdk-version --lang go --no-v-prefix)
            echo "Derived latest Go SDK release version: $go_latest"
          fi
          echo "go_latest=$go_latest" >> $GITHUB_OUTPUT
```

But not a big deal either way

---

I like the suggestions! Cleans up the code a bit